SQLblogmain.php

```diff
@@ -0,0 +1,37 @@
+<?php
+include ('include/SQLProject.php');
+?>
+
+
+<html>
+
+	<head>
+		<title>Lindsay's Blog</title>
+		<link rel='stylesheet' href='maindesign.css' />
+		<link href="https://fonts.googleapis.com/css?family=Lobster|Montserrat" rel="stylesheet">
+	</head>
+
+	<body>
+		<h1 class='header'> Lindsay's Blog </h1>
+		<h2 class='subtitle'>WORDS FROM A GIRL LEARNING HOW TO CODE </h1>
+
+<?php
+
+function PostABlock($blocknumber){
+	$sorter=getSorter();
```

Comment:
Nice job moving the `view_post.php` page into the database. this should be next. Ideally you won't have any of the blog content in the `$sorter` PHP array anymore, it'll all come from the db.